http-client/Network/HTTP/Client/Request.hs

```diff
@@ -201,6 +201,10 @@ getUri req = URI
             _ -> S8.unpack $ queryString req
     , uriFragment = ""
     }
+  where
+    port'
+      | secure req && (port req) == 443 || (port req) == 80 = ""
```

Comment:
Not quite right: this will hide the port for `secure req == True && port req == 80`. Also, personal preference: would you mind either grouping with parents to clarify the operator precedence, or making this two separate guards?

Reply:
Duh, no sure how I could mess this up.

> or making this two separate guards

That is what I did.